.vib/ghost/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,53 @@
+const COMMAND_DELAY = 500;
+const BASE_URL = 'http://bitnami-ghost.my';
+
+for (const command of ['click']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, COMMAND_DELAY);
+    });
+  });
+}
+
```

Comment:
I appreciate the necessity to explain various corner case behaviours, but I'd prefer to keep this data documented internally instead of having them in the comments. Let's create a task for this in our Helm chart strategy and brainstorm different ways of documenting this. 
The cognitive complexity of the code itself is already there for the reader, it is quite difficult to cope with if we add a lot of comments instead.

Reply:
While I agree that it is good having this documented in our strategy, I still see it valuable to have these corner cases in the specific tests where it applies. If this is a workaround that most assets will use then I don't see the point either, but I believe that we won't have that many using `hosts`. In fact, this is the only one that currently has it out of all the others already implemented.

IMHO, I'd rather be too verbose in these workarounds in the code than not commenting at all. Someone that does not have the context will probably don't understand why I am overwriting the built-in `visit` function here.